include/GoogleSync/GoogleSync.php

```diff
@@ -972,9 +1127,8 @@ public function setTimezone($timezone)
      *
      * @return bool Success/Failure
      */
-    protected function setLastSync($event_local, $gEventId = null)
+    protected function setLastSync(Meeting $event_local, $gEventId = null)
```

Comment:
in Line (https://github.com/salesagility/SuiteCRM/pull/6539/files#diff-034e880ea433e2cc2571088f7cbc6620R1137): $return should be a valid SuiteCRM guid but save() method could be return with a failure. Please validate $return variable if it's a valid ID and handle accidental errors before set the accept status as 'accept'.

Reply:
Done